web/pgadmin/static/js/helpers/Notifier.jsx

```diff
@@ -178,7 +178,8 @@ class Notifier {
   confirm(title, text, onOkClick, onCancelClick, okLabel=gettext('Yes'), cancelLabel=gettext('No'), extras=null) {
     /* Use this if you want to use pgAdmin global notifier.
     Or else, if you want to use modal inside iframe only then use ModalProvider eg- query tool */
-    this.modal.confirm(title, text, onOkClick, onCancelClick, okLabel, cancelLabel, extras);
+    const isBtnDanger = (okLabel === 'Delete'); // True if okLabel is "Delete" to change button background color.
```

Comment:
This is wrong, label can be in any language. We already have an `extras` param to pass any other info.